django/db/backends/base/base.py

```diff
@@ -182,6 +182,12 @@ def queries(self):
                 "will be returned.".format(self.queries_log.maxlen))
         return list(self.queries_log)
 
+    def get_database_version(self):
+        raise NotImplementedError('subclasses of BaseDatabaseWrapper may require a get_database_version() method.')
+
+    def is_database_version_supported(self):
+        return self.get_database_version() >= self.features.minimum_database_version
```

Comment:
I would add:
`return self.features.minimum_database_version is None or ....`
so that this method doesn't crash if a minimum version isn't specified.

Reply:
I've added this condition and a test